modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.d.ts

```diff
@@ -10,6 +10,7 @@ export function FieldBase(props: IFieldBase): JSX.Element;
 interface IFieldBase {
 	accessible?: boolean;
 	children?: any;
+	defaultLanguageId?: any;
```

Comment:
Hi @georgel-pop-lr, we avoid using any as a type. Please, replace this line with:
```suggestion
	defaultLanguageId?: Liferay.Language.Locale;
```

Reply:
This is now outdates as I'm not changing any more all this files, the new solution is more simple as I changed all the logic on FormInfo component.

Reply:
Great, @georgel-pop-lr! :smiley: